pre-emails/get_info.py

```
@@ -0,0 +1,340 @@
+        print(f"\n⚠️  Could not auto-download cover image: unsupported URL scheme '{parsed.scheme}'", file=sys.stderr)
+        return ""
+
+    images_dir = Path(__file__).resolve().parent / "images"
+    url_path = url.split("?")[0].rstrip("/")
+    ext = Path(url_path).suffix or ".jpg"
+    filename = Path(url_path).name or f"cover{ext}"
+    local_path = images_dir / filename
+
+    def _print_manual_steps(reason: str) -> None:
+        print(f"\n⚠️  Could not auto-download cover image: {reason}", file=sys.stderr)
+        print(f"\nTo add it manually:", file=sys.stderr)
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)